apps/stable_diffusion/src/utils/stable_args.py

```diff
@@ -164,7 +164,7 @@ def path_expand(s):
 p.add_argument(
     "--hf_model_id",
     type=str,
-    default="stabilityai/stable-diffusion-2-1-base",
+    default="",
```

Comment:
Shouldn't we leave this as default ? For users that don't have any custom ckpt ?  And want a sane default ?

Reply:
The current WebUi code is forcing it to "", so I don't think it will change anything
```
                    hf_model_id = gr.Textbox(
                        placeholder="Select 'None' in the Models dropdown on the left and enter model ID here e.g: SG161222/Realistic_Vision_V1.3",
                        value="",
                        label="HuggingFace Model ID",
                    )
```
But I just noticed the placeholder is refering to an unknown model to me, perhaps updating it with "stabilityai/stable-diffusion-2-1-base" would be preferable ?

Reply:
Yes, you're right !
I'm even having an issue using the command line script and connecting to hf, I didn't noticed it earlier.
I will rework that and test more.